database/seeds/UserTableSeeder.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Faker\Factory as Faker;
+use App\User;
+
+class UserTableSeeder extends Seeder {
+  public function run()
```

Comment:
The below looks very complicated.

Check this out https://laravel.com/docs/5.2/seeding#using-model-factories